cpp/tests/utilities/column_utilities.cu

```diff
@@ -410,7 +413,8 @@ class corresponding_rows_not_equivalent {
                                     table_device_view d_rhs,
                                     column_device_view lhs_row_indices_,
                                     column_device_view rhs_row_indices_,
-                                    size_type fp_ulps_)
+                                    size_type fp_ulps_,
+                                    DeviceComparator /*comp*/)
```

Comment:
A comparator is still used here: https://github.com/rapidsai/cudf/pull/12777/files#diff-4292feecdf2f0106ee3b94779dd191033fd27c494bdffb944685c94fade6fc74R466. That `comp` also needs to be replaced by this `comp`.

Reply:
You are right, in fact the check is not even needed because the caller site only works with 1 column anyway